packages/bot-web-ui/src/components/flyout.jsx

```diff
@@ -11,15 +11,15 @@ import '../assets/sass/flyout.scss';
 
 const SearchResult = ({ search_term, total_result }) => (
     <div className='flyout__search-header'>
-        <span className='flyout__search-header-text'>
+        <Text weight='bold' className='flyout__search-header-text'>
```

Comment:
Should we update the `flyout.scss` file to remove unused styles for this block?

Reply:
You mean ` align-self: center;` ?

Reply:
Nah, that is my bad. I thought we are refactoring all "Text" components, but we are doing only span tags. Please ignore my comment above.